tests/kafkatest/version.py

```diff
@@ -257,6 +257,4 @@ def get_version(node=None):
 V_3_6_0 = KafkaVersion("3.6.0")
 LATEST_3_6 = V_3_6_0
 
-# 3.7.x versions
-V_3_7_0 = KafkaVersion("3.7.0")
-LATEST_3_7 = V_3_7_0
\ No newline at end of file
+# only add released versions here
```

Comment:
Side cleanup -- happens all the time -- added a comment now -- we only add released versions here. There is no 3.7.0 release yet -- seems even 3.6.0 was added pre-mature -- your PR bumping the versions should have added 3.6.0...

Reply:
Thanks for adding this comment. It will clarify for other RMs or whoever is making the required changes for the release.